frameworks/java/prewarm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "loading" > /workspace/prewarm.status
+
+# prewarm by starting the gradle daemon. Running an initial test build will also speed things up a bit
+cd /runner/frameworks/java && gradle --daemon --offline test
```

Comment:
You can also start monitoring the memory usage of the container by
```
docker stats
# or
# docker stats --format "table {{.MemUsage}}"
```
and creating/using the container in another terminal session.

This will stream something like this (not running, started, doing some work, stopped):
```
CONTAINER           CPU %               MEM USAGE / LIMIT   MEM %               NET I/O             BLOCK I/O           PIDS
CONTAINER           CPU %               MEM USAGE / LIMIT   MEM %               NET I/O             BLOCK I/O           PIDS
e9973a8299eb        0.00%               0B / 0B             0.00%               0B / 0B             0B / 0B             0
CONTAINER           CPU %               MEM USAGE / LIMIT   MEM %               NET I/O             BLOCK I/O           PIDS
e9973a8299eb        0.12%               40.34MiB / 1GiB     3.94%               168B / 0B           0B / 0B             18
CONTAINER           CPU %               MEM USAGE / LIMIT   MEM %               NET I/O             BLOCK I/O           PIDS
```